resources/js/features/meeting/DeleteMeeting.tsx

```diff
@@ -12,17 +12,17 @@ export async function confirmDelete(
   t: LangFunc,
   meeting: MeetingForDelete,
 ): Promise<SeriesDeleteOption | undefined> {
-  const [deleteOption, setDeleteOption] = createSignal(SeriesDeleteOption.ONE);
+  const [deleteOption, setDeleteOption] = createSignal<SeriesDeleteOption>("one");
   const isSelected = createSelector(deleteOption);
```

Comment:
O kurcze, nigdy tego nie użyłem :) To ma sens zdaje się jak jest długa lista i w ogóle, w celach optymalizacyjnych, a tutaj czytelniej byłoby `deleteOption()==="one"`. No ale pewnie może być.